contracts/tasks/aero.js

```diff
@@ -0,0 +1,162 @@
+  const vault = await resolveContract("OETHBaseVaultProxy", "IVault");
+  const oeth = await resolveContract("OETHBaseProxy", "OETHBase");
+  const pool = await resolveContract(base.aerodromeOETHbWETHClPool, "ICLPool");
+  const aeroStrat = await resolveContract(
+    `AerodromeAMOStrategyProxy`,
+    "AerodromeAMOStrategy"
+  );
+  const sugarHelper = await resolveContract(base.sugarHelper, "ISugarHelper");
+
+  const Q96 = BigNumber.from(2).pow(96);
+  const sqrtRatioX96TickLower = BigNumber.from("79224201403219477170569942574"); // -1 tick
+  const sqrtRatioX96TickHigher = BigNumber.from(
+    "79228162514264337593543950336"
+  ); // 0 tick
+
+  const { tick, sqrtPriceX96 } = await pool.connect(signer).slot0({ blockTag });
+  const { liquidityGross } = await pool.connect(signer).ticks(-1, { blockTag });
+  const { amount0: tickWethBalance, amount1: tickOethBalance } =
+    await sugarHelper
+      .connect(signer)
+      .getAmountsForLiquidity(
+        sqrtPriceX96,
+        sqrtRatioX96TickLower,
+        sqrtRatioX96TickHigher,
+        liquidityGross,
+        { blockTag }
+      );
+
+  // Pool balances
+  const poolPrice = sqrtPriceX96
+    .mul(sqrtPriceX96)
+    .mul(10000000000)
+    .div(Q96)
+    .div(Q96);
+  const poolWethBalance = await weth
+    .connect(signer)
+    .balanceOf(base.aerodromeOETHbWETHClPool, { blockTag });
+  const poolOethBalance = await oeth
+    .connect(signer)
+    .balanceOf(base.aerodromeOETHbWETHClPool, { blockTag });
+  const poolTotal = poolWethBalance.add(poolOethBalance);
+  const poolWethPercentage = poolWethBalance.mul(10000).div(poolTotal);
+  const poolOethPercentage = poolOethBalance.mul(10000).div(poolTotal);
+
+  // Tick balances
+  const tickTotal = tickWethBalance.add(tickOethBalance);
+  const tickWethPercentage = tickWethBalance.mul(10000).div(tickTotal);
+  const tickOethPercentage = tickOethBalance.mul(10000).div(tickTotal);
+  const tickTotalPercentage = tickTotal.mul(10000).div(poolTotal);
+
+  // Strategy's tick position
+  const {
+    _amountWeth: tickStratWethBalance,
+    _amountOethb: tickStratOethBalance,
+  } = await aeroStrat.getPositionPrincipal();
+  const tickStratTotal = tickStratWethBalance.add(tickStratOethBalance);
+  const tickStratWethPercentage = tickStratWethBalance
+    .mul(10000)
+    .div(tickStratTotal);
+  const tickStratOethPercentage = tickStratOethBalance
+    .mul(10000)
+    .div(tickStratTotal);
+  const tickStratTotalOfTickPercentage = tickStratTotal
+    .mul(10000)
+    .div(tickTotal);
+  const tickStratTotalOfPoolPercentage = tickStratTotal
+    .mul(10000)
+    .div(poolTotal);
+
+  const checkBalance = await aeroStrat
+    .connect(signer)
+    .checkBalance(weth.address, { blockTag });
+
+  const vaultWethBalance = await weth
+    .connect(signer)
+    .balanceOf(vault.address, { blockTag });
+
+  // Pool balances
+  console.log(
+    `Pool price       : ${formatUnits(poolPrice, 10)} OETHb/WETH, ${tick} tick`
+  );
+  console.log(
```

Comment:
I think these are fine, maybe there should be just additional notice, that these balances include unclaimed WETH/OETH from swap fees and liquidity decreases.

Reply:
I've added a comment